addon/components/pix-modal.js

```diff
@@ -20,4 +21,8 @@ export default class PixModal extends Component {
   isClickOnOverlay(event) {
     return event.target.classList.contains('pix-modal__overlay');
   }
+
+  get id() {
+    return uniqueId;
```

Comment:
ça devrait être mieux comme ça ;) 
```suggestion
    return uniqueId();
```